kubectl-plugin/test/e2e/testdata/rayjob-submit-working-dir/entrypoint-python-sample.py

```diff
@@ -0,0 +1,28 @@
+import ray
+import os
+import requests
+
+ray.init()
+
+@ray.remote
+class Counter:
+    def __init__(self):
+        # Used to verify runtimeEnv
+        self.name = os.getenv("counter_name")
+        assert self.name == "test_counter"
+        self.counter = 0
+
+    def inc(self):
+        self.counter += 1
+
+    def get_counter(self):
+        return "{} got {}".format(self.name, self.counter)
+
+counter = Counter.remote()
+
+for _ in range(5):
+    ray.get(counter.inc.remote())
+    print(ray.get(counter.get_counter.remote()))
```

Comment:
I think use task would be simpler than actor. We don't need counter. A simple task that checks environment variables and try to import package in it is enough. Something like

```python
import ray

ray.init()

@ray.remote
def f():
    # import package
    # check package version
    # check env vars

ray.get(f.remote())
```

Reply:
PTAL, I used 2 packages that aren't dependencies and 2 env vars.